pkg/controlloop/pod.go

```diff
@@ -33,11 +33,12 @@ import (
 )
 
 const (
-	defaultMountPath      = "/host"
-	ipReconcilerQueueName = "pod-updates"
-	syncPeriod            = time.Second
-	whereaboutsConfigPath = "/etc/cni/net.d/whereabouts.d/whereabouts.conf"
-	maxRetries            = 2
+	defaultMountPath         = "/host"
+	ipReconcilerQueueName    = "pod-updates"
+	syncPeriod               = time.Second
+	whereaboutsConfigPath    = "/etc/cni/net.d/whereabouts.d/whereabouts.conf"
+	maxRetries               = 2
+	defaultReconcilerTimeout = 30 // added for reconciler invocation
```

Comment:
isn't this the exact same thing as in https://github.com/k8snetworkplumbingwg/whereabouts/pull/238/commits/6035bb030bee3c6caed052779d7b1fa72ab1dead#diff-aa6e374b725fa31fa303df2c2516ff46f2d936054e1d8de78605a48c07d8a2dcR11 ?

Please stick to one if possible. Is there a reason for duplicating them ?

Reply:
Yes it is the same exact thing. Looks like an oversight, I don't even see it referenced in this file. Good catch!